### PR TITLE
[New Spritepack] Jurassic T-Shirts

### PR DESCRIPTION
**Pull request by kkrosie123**
- [x] jurassic world t-shirt
- [x] jurassic park t-shirt

### CODE REVIEW

**Comment by kkrosie123** [comment]
it's ready, just needs thumbs

File: spritepacks/released/briaryoung/briaryoung-tshirt-jurassic-park/mod_assets/monika/j/briaryoung_clothes_tshirt_jurassic_park.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+    "__author": "Briar Young",
+    "ex_props": {
+        "bare-shoulders": true,
+        "home": true,
+        "date": true
```
Comment (ThePotatoGuy) [inline spritepacks/released/briaryoung/briaryoung-tshirt-jurassic-park/mod_assets/monika/j/briaryoung_clothes_tshirt_jurassic_park.json:6]: are these exprops correct? bare-shoulders is probably wrong, but would you wear a jurassic park T-shirt on a date?

File: spritepacks/released/briaryoung/briaryoung-tshirt-jurassic-world/mod_assets/monika/j/briaryoung_clothes_tshirt_jurassic_world.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
+{
+    "__author": "Briar Young",
+    "ex_props": {
+        "bare-shoulders": true,
+        "home": true,
+        "date": true
```
Comment (ThePotatoGuy) [inline spritepacks/released/briaryoung/briaryoung-tshirt-jurassic-world/mod_assets/monika/j/briaryoung_clothes_tshirt_jurassic_world.json:6]: same exprops issues here

**Review by ThePotatoGuy — CHANGES_REQUESTED** [review]

File: spritepacks/released/briaryoung/briaryoung-tshirt-jurassic-park/mod_assets/monika/j/briaryoung_clothes_tshirt_jurassic_park.json (hunk 1 above)
Comment (kkrosie123) [reply spritepacks/released/briaryoung/briaryoung-tshirt-jurassic-park/mod_assets/monika/j/briaryoung_clothes_tshirt_jurassic_park.json:6]: Hah. Oops. I stole this from another `json` and forgot to change exprops.

**Comment by jmwall24** [comment]
assigning @Legendkiller21 for thumbs

**Review by multimokia — APPROVED** [review]

**Review by jmwall24 — APPROVED** [review]

**Review by ThePotatoGuy — APPROVED** [review]

**Pull request merged**